log_injection/BS-World_Google-Ads-auto-clicker/ip.py

```
@@ -0,0 +1,30 @@
+import subprocess
+import time
+import logging
+
+def change_ip(interface, new_ip, subnet_mask='255.255.255.0', gateway='1.1.1.1'):
+    try:
+        # Command to change IP address
+        command = [
+            'netsh', 'interface', 'ip', 'set', 'address',
+            interface, 'static', new_ip, subnet_mask, gateway
+        ]
+        subprocess.run(command, check=True)
```

Ruff (0.13.3)

`subprocess` call: check for execution of untrusted input

(S603)